www/perl-lib/SMT/SCCSync.pm

```diff
@@ -1607,7 +1608,7 @@ sub _updateProductData
         printLog($self->{LOG}, $self->vblevel(), LOG_ERROR, "Cannot connect to database.");
         return 1;
     }
-    my $nuurl = URI->new($self->{CFG}->val("NU", "NUUrl", "https://updates.suse.com/"));
+    my $nuurl = URI->new($self->{CFG}->val("NU", "NUUrl", "https://dl.suse.com/"));
```

Comment:
This is the default value taken when no `NUUrl` is set when an SMT gets configured without setting this attribute in the config. This mean it only affects new installations which configure SMT **without** the wizard **and** do not set this value.

Reply:
Ok, maybe add this as a comment here for future us?